documentation/api/query/v4/catalogs.markdown

```diff
@@ -21,38 +21,37 @@ You can query catalogs by making an HTTP request to the
 
 This will return a JSON array containing the most recent catalog for each node in your infrastructure.
 
-### URL Parameters
-* `query`: Optional. A JSON array containing the query in prefix notation (`["<OPERATOR>", "<FIELD>", "<VALUE>"]`). See the sections below for the supported operators and fields. For general info about queries, see [the page on query structure.][query]
+### URL parameters
+* `query`: optional. A JSON array containing the query in prefix notation (`["<OPERATOR>", "<FIELD>", "<VALUE>"]`). See the sections below for the supported operators and fields. For general info about queries, see [our guide to query structure.][query]
 
 If a query parameter is not provided, all results will be returned.
 
-### Query Operators
+### Query operators
 
-See [the Operators page.](./operators.html)
+See [the query operators page.](./operators.html)
 
-### Query Fields
+### Query fields
 
-* `certname` (string): the certname associated with the catalog
-* `version` (string): an arbitrary string that uniquely identifies each catalog for a node
-* `environment` (string): the environment associated with the catalog's certname
-* `transaction_uuid` (string): a string used to tie a catalog to a report from the same puppet run
-* `code_id` (string): a string used to tie a catalog to the Puppet code which generated the catalog
-* `hash` (string): sha1 hash of the resources of associated with a node's most
-  recent catalog
+* `certname` (string): the certname associated with the catalog.
+* `version` (string): an arbitrary string that uniquely identifies each catalog for a node.
+* `environment` (string): the environment associated with the catalog's certname.
+* `transaction_uuid` (string): a string used to tie a catalog to a report from the same Puppet run.
+* `code_id` (string): a string used to tie a catalog to the Puppet code which generated the catalog.
+* `hash` (string): SHA-1 hash of the resources of associated with a node's most
+  recent catalog.
 * `producer_timestamp` (string): a string representing the time at which the
   `replace_catalog` command for a given catalog was submitted from the master.
 
-### Subquery Relationships
+### Subquery relationships
 
-Here is a list of related entities that can be used to constrain the result set using
-implicit subqueries. For more information consult the documentation for [subqueries].
+The following list contains related entities that can be used to constrain the result set using implicit subqueries. For more information consult the documentation for [subqueries][subqueries].
```

Comment:
we're inconsistent about line length in our markdown, but in general I don't like joining distinct lines like this because it makes actual changes harder to track. Just a note, not a suggestion to go and revert these cases.